pkg/sfu/turn.go

```diff
@@ -0,0 +1,170 @@
+package sfu
```

Comment:
Should not be needed I think.

Reply:
reference by  sfu.go->session.go->router.go